### PR TITLE
Jc admin video

### PR DESCRIPTION
**Pull request by carriewalsh**
Tackled admin tutorial edit problems:
- enabled admin to add tutorial
- fixed edit video workflow
- fixed small syntax issues for params
- added tests for video updating

GithubService update
- adds current_user argument to spec files

### CODE REVIEW

File: spec/services/github_service_spec.rb [hunk 1]
```diff
@@ -2,30 +2,31 @@
...
   context "instance methods", :vcr do
-    it "gets repos" do
-      service = GithubService.new
+    before :each do
+      user = create(:user, access_token: ENV["GITHUB_TOKEN_KEY"])
+      @service = GithubService.new(user)
+    end
 
-      result = service.get_repos
+    it "gets repos" do
```
[the remark this answers is not in the document]
Comment (jalena-penaligon) [reply spec/services/github_service_spec.rb]: Resolved syntax issues.

File: app/controllers/admin/videos_controller.rb [hunk 2]
```diff
@@ -25,11 +25,11 @@ def create
   end
 
   private
-    def video_params
-      params.permit(:position)
+    def new_video_params
```
[the remark this answers is not in the document]
Comment (jalena-penaligon) [reply app/controllers/admin/videos_controller.rb:28]: Private methods intentionally tabbed in for code readability.

File: app/controllers/admin/videos_controller.rb [hunk 3]
```diff
@@ -25,11 +25,11 @@ def create
...
+      params.require(:video).permit(:title, :description, :video_id, :position)
     end
 
-    def new_video_params
+    def video_params
```
[the remark this answers is not in the document]
Comment (jalena-penaligon) [reply app/controllers/admin/videos_controller.rb:32]: Private methods intentionally tabbed in for code readability.

**Comment by erin-king** [comment]
Looks good- Jalena, thank you for resolving the HoundCI issues- it really doesn't like the double quotes. Good job on finding the naming discrepancies and resolving the admin video problems.

**Pull request merged**